src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorDefinitionProvider.ts

```diff
@@ -17,7 +17,7 @@ export class RazorDefinitionProvider
         token: vscode.CancellationToken) {
 
         const projection = await this.getProjection(document, position, token);
-        if (!projection || projection.languageKind === LanguageKind.Razor) {
```

Comment:
I wouldn't imagine you'd need to do this for your bits to work

Reply:
Wasn't working before :(. I suspect it's a similar issue to the rename stuff.

Reply:
That seems concerning. Was it returning C# in this instance?

Reply:
Oh I just changed it back and it still works, so perhaps something else was broken while I was trying this! Should be good now.